### DISCUSSION TITLE
Limiting organization members to specific projects

### DISCUSSION THREAD

**Discussion by bombillazo** [supabase/supabase#18927] [Feature Requests] [answered]
For security reasons, we require team members in our organization to be assigned roles **AND** project access since a member is not assumed they can access any and all projects in the organization. This creates a significant security risk for our product dev process since we have to limit our interactions with external devs or clients across the organization.

**Comment by Akash187** [comment]
Yes, this is a serious issue.
Also, all developers get access to **service_role** key which I am not very comfortable with.

**Comment by w3b6x9** [comment, marked as answer]
we've introduced new platform access controls that provide more granular permissions for projects: https://supabase.com/blog/platform-access-control

**Reply by Akash187** [reply]
Thanks @w3b6x9 for the update liked it. I would have appreciated `read-only` permission for Pro Plans. The Team plan is too expensive for such a feature.

**Reply by bombillazo** [reply]
Happy to hear this, hopefully we can test it soon
